experimental/dds/tree2/src/test/shared-tree-core/editManager.spec.ts

```diff
@@ -226,7 +226,7 @@ describe("EditManager", () => {
 			{ seq: 4, type: "Pull", ref: 1, from: peer2 },
 			{ seq: 6, type: "Pull", ref: 3, from: peer1 },
 			{ seq: 8, type: "Pull", ref: 3, from: peer1 },
-			{ seq: 10, type: "Pull", ref: 0, from: peer2 },
+			{ seq: 10, type: "Pull", ref: 1, from: peer2 },
```

Comment:
This was likely a mistake; note that the previous value (0) is less than the ref value received from peer2 at step 2 (1). It should be impossible for a reference sequence number to ever decrease.